### DISCUSSION TITLE
How can I attach additional info to the ID token?

### DISCUSSION THREAD

**Discussion by maximgx** [fief-dev/fief#366] [Q&A]
In the architecture of my app a `user` can be part of multiple `organisation`s.

I want to attach to the [ID token](https://docs.fief.dev/going-further/id-token-encryption/) (or something else) at least the name of the organisation it authenticated in – so that multiple microservices know what org this user is part of.

How can I achieve this?

It seems that I cannot use the standard `org:read/write` that fief provides because it's a dynamic list of organisations.

**Comment by fief-bailiff** [comment]
Hail, @maximgx 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to [star our repository](https://github.com/fief-dev/fief) ⭐️

<p align="center">
<img width="656" alt="star-fief" src="https://github.com/fief-dev/fief/assets/1144727/b2db7f59-102f-47e5-9751-67fa10372c52">
</p>

## Want to support us?

Subscribe to one of our paid plan to help us continue our work and receive exclusive information and benefits! Starts at $5/month 🪙

<p align="center">
    <a href="https://polar.sh/fief-dev">
    <picture>
      <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/embed/subscribe.svg?org=fief-dev&darkmode=1">
      <img alt="Subscribe" src="https://polar.sh/embed/subscribe.svg?org=fief-dev">
    </picture>
    </a>
</p>

Farewell!
